src/Main.java

```diff
@@ -1,5 +1,74 @@
 public class Main {
+
+    public static final Employee[] employees = {
```

Comment:
поля static И final надо называть в верхнем регистре, т.е.
```suggestion
    public static final Employee[] EMPLOYEES = {
```

Reply:
также поле лучше сделать приватным